### ISSUE TITLE
Preview does not show image

### ISSUE THREAD

**Issue by rohel01** [qjebbs/vscode-plantuml#129]
Hello,

I am using latest vscode/plantuml extension on Ubuntu 16.04.
I have installed graphviz  2.38 through apt.

The extension works fine, except for the live preview. I get the following error log

[hs_err_pid26718.log](https://github.com/qjebbs/vscode-plantuml/files/2191821/hs_err_pid26718.log)

**Comment by qjebbs** [comment]
I can tell it's a problem of the environment (OpenJDK?) instead of the extension, see explains here: https://github.com/qjebbs/vscode-plantuml/issues/127

**Comment by rohel01** [comment]
By default, java 9 is used on my machine.
If I use the `"plantuml.java"` setting to point to java 8 instead, it works

**Comment by rohel01** [comment]
Yes. You still bundle a plantuml JAR. Documenting this limitation would prevent a lot of noise imo

**Comment by qjebbs** [comment]
Yes. Thanks for the advice.
